lib/simplecov_lcov_formatter.rb

```diff
@@ -1,8 +1,8 @@
 require 'fileutils'
 require 'pathname'
-require_relative 'simple_cov_lcov/configuration'
+require 'simplecov'
 
-fail 'simplecov-lcov requires simplecov' unless defined?(SimpleCov)
```

Comment:
This check is no longer required, because gemspec needs 'simplecov' on installed environment.

Strictly, this message is still efficient if users forget to `require 'simplecov'` on their code. But this is not critical I think.